src/custom/components/swap/EthWethWrap/helpers.ts

```diff
@@ -0,0 +1,30 @@
+import { parseUnits } from 'ethers/lib/utils'
+import { CurrencyAmount } from '@uniswap/sdk'
+import { DEFAULT_PRECISION } from 'constants/index'
+
+export const MINIMUM_TXS = '10'
+export const AVG_APPROVE_COST_GWEI = '50000'
+export const DEFAULT_GAS_FEE = parseUnits('50', 'gwei')
+
+export const setNativeLowBalanceError = (nativeSymbol: string, lowBalanceThreshold: CurrencyAmount) =>
+  new Error(
+    `WARNING! After wrapping your ${nativeSymbol}, your balance will fall below < ${lowBalanceThreshold.toSignificant(
+      DEFAULT_PRECISION
+    )} ${nativeSymbol}. As a result you may not have sufficient ${nativeSymbol} left to cover future on-chain transaction costs.`
+  )
+
+export function checkUserBalance({
```

Comment:
Since it returns a boolean, could you rename it so it expresses what's the result?
Right now I don't know whether `true` is good or bad.

Maybe `isBalanceEnough` or `shouldDisplayLowBalanceWarning`

Reply:
:+1: